fog/ledger/enclave/impl/src/lib.rs

```diff
@@ -7,30 +7,60 @@
 //! combined with a form of dependency injection, would provide the machines
 //! with all the unit testing they would ever need. Fate, it seems, has a sense
 //! of irony...
-
 #![no_std]
-
```

Comment:
can we please add `#![deny(missing_docs)]` to this crate and add required doc strings?